package.json

```diff
@@ -71,6 +71,7 @@
     "test": "jest",
     "test-run": "export JEST_HTML_REPORTERS_PUBLIC_PATH=reports/html-report-seq; rm nonce.db ; rm sudo.lock;  node --experimental-specifier-resolution=node --loader ts-node/esm --experimental-vm-modules node_modules/jest/bin/jest.js --forceExit ",
     "test-parallel": "export JEST_HTML_REPORTERS_PUBLIC_PATH=reports/html-report-parallel;rm nonce.db ; rm sudo.lock; node --expose-gc --experimental-specifier-resolution=node --loader ts-node/esm --experimental-vm-modules node_modules/jest/bin/jest.js --verbose --ci --group=parallel --forceExit --workerIdleMemoryLimit=200MB --logHeapUsage",
+    "test-parallel-3rdPartyRewards": "export JEST_HTML_REPORTERS_PUBLIC_PATH=reports/html-report-parallel;rm nonce.db ; rm sudo.lock; node --expose-gc --experimental-specifier-resolution=node --loader ts-node/esm --experimental-vm-modules node_modules/jest/bin/jest.js --verbose --ci --group=3rdPartyRewards --forceExit",
```

Comment:
As we added these tests to `sequential` folder we should add  `--runInBand` here

Reply:
Hum, actually they must be moved to parallel!